### PR TITLE
fix(Instagram): profile element changed

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Changed the profile element and changed the way it's used due to different formats
## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/I/Instagram/presence.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ presence.on("UpdateData", async () => {
 				.querySelector("head > title")
 				?.textContent.split("(")[0]
 				.trim() ?? "Unknown"
-		} (@${profileName.textContent})`;
+		} (${profileName[1].split(")")[0]})`;
```
Comment (EncryptedDev) [inline websites/I/Instagram/presence.ts:100]: Why remove the `@`

Comment (DarkVillager) [reply websites/I/Instagram/presence.ts:100]: its already present in the new element

**Review by EncryptedDev — APPROVED** [review]

File: websites/I/Instagram/presence.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ presence.on("UpdateData", async () => {
 		]),
 		{ pathname } = window.location,
 		path = pathname.split("/"),
-		profileName = document.querySelector("div.XBGH5 h2");
+		profileName = document.title.split("(");
```
Comment (Bas950) [inline websites/I/Instagram/presence.ts:17]:
```suggestion
		[,profileName] = document.title.split("(");
```

File: websites/I/Instagram/presence.ts [hunk 3]
```diff
@@ -87,7 +87,7 @@ presence.on("UpdateData", async () => {
 		pathname.startsWith("/direct/t")
 	)
 		presenceData.details = "Direct Messages";
-	else if (profileName?.textContent === path[1]) {
+	else if (profileName[1].split(")")[0].replace("@", "") === path[1]) {
```
Comment (Bas950) [inline websites/I/Instagram/presence.ts:90]:
```suggestion
	else if (profileName.split(")")[0].replace("@", "") === path[1]) {
```

File: websites/I/Instagram/presence.ts (hunk 1 above)
Comment (Bas950) [inline websites/I/Instagram/presence.ts:100]:
```suggestion
		} (${profileName.split(")")[0]})`;
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**